singularity.spec.in

```diff
@@ -139,11 +134,14 @@ rm -rf $RPM_BUILD_ROOT
 %{_libexecdir}/singularity/cli/shell.*
 %{_libexecdir}/singularity/cli/test.*
 %{_libexecdir}/singularity/bin/action
+%{_libexecdir}/singularity/bin/cleanupd
 %{_libexecdir}/singularity/bin/start
 %{_libexecdir}/singularity/bin/docker-extract
+%{_libexecdir}/singularity/bootstrap-scripts
```

Comment:
I'm not too sure about including bootstrap-scripts/* in the runtime rpm, since they are scripts for bootstrapping while building containers, which is not really runtime functionality. I understand that some of this is needed to get a docker container working though... see main review comment.